lib/index.js

```diff
@@ -5,17 +5,18 @@ const CkanAuthApi = require('./util/ckan-auth-api')
 const CkanUploadAPI = require('./util/ckan-upload-api')
 const ActionApi = require('./util/action-api')
 const Open = require('./file')
+const { camelToSnakeCase } = require('./util/general')
 
 /**
  * @class Client.
- * @param {string} authToken
+ * @param {string} apiKey
  * @param {number} organizationId
  * @param {string} datasetId
  * @param {string} api
  */
 class Client {
-  constructor(authToken, organizationId, datasetId, api) {
-    this.authToken = authToken
+  constructor(apiKey, organizationId, datasetId, api) {
```

Comment:
great we have this now - though to be clear my previous comment was an aside and if we change we should probably do in a separate commit to avoid breaking anything and clean PRs.

Reply:
👍